crates/swift-bridge-ir/src/codegen/generate_rust_tokens.rs

```diff
@@ -275,12 +280,17 @@ impl ToTokens for SwiftBridgeModule {
                 }
             };
         }
-
+        let custom_type_definitions: Vec<TokenStream> = custom_type_definitions
```

Comment:
This is fine, but `.into_values()` would also work here.

i.e.

```rust
let custom_type_definitions = custom_type_definitions.into_values();
let module_inner = quote! { 
    /* ... */ 

    #(#custom_type_definitions)*
}
```

should work.

---

Just a heads up in case you didn't know.